mongomock/filtering.py

```diff
@@ -265,7 +265,7 @@ def _iter_key_candidates_sublist(key, doc):
 
 
 def _force_list(v):
-    return v if isinstance(v, (list, tuple)) else [v]
+    return v if isinstance(v, (list, tuple)) and len(v) > 0 else [v]
```

Comment:
Please use the more pythonic boolean comparison `return v if v and isinstance(v, (list, tuple)) else [v]`